common/src/config.rs

```diff
@@ -252,6 +255,8 @@ struct Cache {
     relay_expiry: u32,
     /// The cache timeout for events (store) before dropping them.
     event_expiry: u32,
+    /// The maximum amount of events to queue before dropping them.
+    event_buffer_size: u32,
```

Comment:
That's for later, but we should introduce a similar setting for the project cache.